orchestration/agent_monitor.py

```
@@ -170,7 +175,7 @@ def get_workspace_modified_time(self, agent_name: str):
                                     mtime = os.path.getmtime(file_path)
                                     if mtime > latest_time:
                                         latest_time = mtime
-                                except OSError:
+                                except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)